SPECS-EXTENDED/hunspell/hunspell.spec

```
@@ -3,11 +3,11 @@
 Name:      hunspell
 Summary:   A spell checker and morphological analyzer library
 Version:   1.7.0
-Release:   6%{?dist}
+Release:   7%{?dist}
 # Source:  https://github.com/hunspell/hunspell/archive/refs/tags/v1.7.0.tar.gz
-Source:    https://github.com/hunspell/hunspell/archive/refs/tags/%{name}-%{version}.tar.gz
+Source0:  https://github.com/hunspell/hunspell/archive/refs/tags/v%{version}.tar.gz#/%{name}-%{version}.tar.gz
 URL:       https://github.com/hunspell/hunspell
-License:   LGPLv2+ or GPLv2+ or MPLv1.1
+License:   LGPLv2+ or GPLv2+ or MPLv1.1 or BSD
```

Thinking more about it - this might need to be the following:

```suggestion
License:   (LGPLv2+ or GPLv2+ or MPLv1.1) and BSD
```

I might be wrong, but here's my take- given that the original myspell source is BSD, BSD always applies. The changes from the hunspell team are licensed under the tri-license thing. Thus, we get the license with the above grouping. @PawelWMS, thoughts?

---

Yes, it sounds reasonable. Not an expert in legal issues but that's how I'd be looking at it as well.